lib/ExpensiMark.js

```diff
@@ -260,8 +260,9 @@ export default class ExpensiMark {
                 name: 'anchor',
                 regex: /<(a)[^><]*href\s*=\s*(['"])(.*?)\2(?:".*?"|'.*?'|[^'"><])*>(.*?)<\/\1>(?![^<]*(<\/pre>|<\/code>))/gi,
                 replacement: (match, g1, g2, g3, g4) => {
-                    const link = g3.startsWith('mailto:') ? g3.slice(7) : g3;
-                    if (g3.startsWith('mailto:') && link === g4) {
+                    const isEmail = g3.startsWith('mailto:');
+                    const link = isEmail ? g3.slice(7) : g3;
+                    if (isEmail && link === g4) {
                         return link;
                     }
```

Comment:
Why do we need to compute `link`, `isEmail` unnecessary computations and variables?

I already pointed it out [here](https://github.com/Expensify/App/issues/9662#issuecomment-1230714765)
>  I **like the idea computing of email instead of link here. Only email linking that doesn't follow markdown link syntax, we have to return email as it is. In all other cases**, we have to format it in markdown link syntax. 

Inferring from the above comments. We could simplify as written below!

```js
const email = g3.startsWith('mailto:') ? g3.slice(7) : '';
if (email === g4) {
    return email;
}
return `[${g4}](${g3})`;
```

@b1tjoy this should do, can you verify and update the changes?

cc: @luacmartins

Reply:
@luacmartins Any thoughts?

Reply:
Seems like this one was resolved?

Reply:
Yes, it was resolved are we good with this?

Reply:
If then let's go ahead and merge, as it looks good & tests pass. We can fully test it in PR against E/App.

@luacmartins